<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartRooms</class>
 <widget class="QMainWindow" name="SmartRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartRooms</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeMono</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Smart Rooms</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>201</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>FreeSans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
         </property>
         <property name="text">
          <string>Define Areas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>FreeSans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton{
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
         </property>
         <property name="text">
          <string>Simulation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="font">
          <font>
           <family>FreeSans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
         </property>
         <property name="text">
          <string>Add Persons</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="areawindow">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeMono</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Define Areas</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>199</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>131</y>
        <width>267</width>
        <height>88</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditRoom">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Area Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRoom_2">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Person IDs</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="simulation">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeMono</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Simulation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>199</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
      </property>
      <property name="text">
       <string>Access Area</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="persons">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeMono</family>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>Add Persons</string>
      </property>
     </widget>
     <widget class="QPushButton" name="homeButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>FreeSans</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>271</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEditRoom_3">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRoom_4">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRoom_5">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditRoom_6">
         <property name="font">
          <font>
           <family>FreeMono</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>IMG Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <family>FreeSans</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton {
border: 1px solid white;
color:white;
}

QPushButton:hover:!pressed
{
  border: 2px solid white;
}</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
